cosmos/carbon.json

```diff
@@ -315,9 +315,9 @@
       "coinDecimals": 6,
       "coinGeckoId": "usd-coin",
       "gasPriceStep": {
-        "low": 0.07692307692307693,
-        "average": 0.09615384615384616,
-        "high": 0.11538461538461539
+        "low": 0.033333333333,
+        "average": 0.033333333333,
+        "high": 0.033333333333
```

Comment:
Same issue as comment on L104, need to round ceil, if not user will be paying insufficient gas.